src/ucs/config/parser.c

```diff
@@ -1156,9 +1156,9 @@ ucs_status_t ucs_config_parser_get_value(void *opts, ucs_config_field_t *fields,
 
         name_len = strlen(field->name);
 
-        ucs_trace("compare name \"%s\" with field \"%s\" which is%s subtable",
+        ucs_trace("compare name \"%s\" with field \"%s\" which is %s subtable",
                   name, field->name,
-                  ucs_config_is_table_field(field) ? "" : " NOT");
+                  ucs_config_is_table_field(field) ? "a" : "NOT a");
```

Comment:
"a"  can be part of constant "a subtable"

Reply:
this is cosmetic change done to have "is%s" -> "is %s"